uaclient/cli/fix.py

```diff
@@ -0,0 +1,866 @@
+import re
+import textwrap
+from typing import Dict, List, NamedTuple, Optional, Set, Tuple, Union  # noqa
```

Comment:
Could we specify the `noqa`? I guess it is unused import, but it's better to make it explicit;
Same for the one below